content/en/examples/ingress/example-ingress.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: networking.k8s.io/v1beta1
```

Comment:
This should now be:
```suggestion
 apiVersion: networking.k8s.io/v1
```

Reply:
Valid point.
But I don't think we should put a blocker here because it is a different issue.
We can file a different issue on fixing the version used for Ingress.

Reply:
In my pr i have not changed this examples. but if you change the api version the rules also should be changed. it has a new struct.

Reply:
I would update the content from the examples.

Reply:
Done.
@sftim @tengqm @miry